static/js/brand-store/Members/MembersTable.js

```diff
@@ -1,8 +1,61 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import { MainTable, CheckboxInput } from "@canonical/react-components";
 
-function MembersTable({ filteredMembers }) {
+function MembersTable({ filteredMembers, changedMembers, setChangedMembers }) {
+  const [members, setMembers] = useState(filteredMembers);
+
+  const handleRoleChange = (member, role) => {
```

Comment:
If you change `member` here to something like `currentMember` then below you can change `item` to `member` and it'll read better :)

Reply:
Done